src/InputContainer.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  updateRestaurant,
+  addRestaurant,
+} from './actions';
+
+import Input from './Input';
+
+export default function InputContainer() {
+  const { restaurant } = useSelector((state) => ({
+    restaurant: state.restaurant,
+  }));
+
+  const dispatch = useDispatch();
+
+  function handleChange(event) {
+    dispatch(updateRestaurant(event.target.name, event.target.value));
+  }
```

Comment:
```suggestion
function handleChange(event) {
    const { target: { name, value } } = event;
    dispatch(updateRestaurant(name, value));
  }
```

말씀하신 에러는 이렇게 하면 해결될 것 같아요~

Reply:
에러를 검색해보니 Input 컴포넌트의 초기값이 설정이 되지 않아서 발생하는 문제처럼 설명들을 하던데, 조언해주신 대로 수정을 해도 warning이 없어지지를 않네요.